casadm/cas_main.c

```diff
@@ -741,6 +752,18 @@ int set_param_seq_cutoff_handle_option(char *opt, const char **arg)
 			cas_printf(LOG_ERR, "Error: Invalid policy name.\n");
 			return FAILURE;
 		}
+	} else if (!strcmp(opt, "promotion-threshold")) {
+		if (validate_str_num(arg[0], "sequential cutoff promotion threshold", 0,
+				4194181) == FAILURE)
+			return FAILURE;
+
+		SET_CORE_PARAM(core_param_seq_cutoff_promotion_threshold, atoi(arg[0]) * KiB);
+	} else if (!strcmp(opt, "promotion-count")) {
+		if (validate_str_num(arg[0], "sequential cutoff promotion request count", 0,
+				65535) == FAILURE)
```

Comment:
why not use UINT32_MAX?

Reply:
Per-stream request count is 16 bits long. We could technically make it bigger, but 2^16 is more than enough for promotion count, which will typically be set to value in the order of tens.